typegraph/core/wit/typegraph.wit

```diff
@@ -18,6 +18,7 @@ interface core {
 
     record type-base {
         name: option<string>,
+        runtime-config: option<list<tuple<string, string>>>
```

Comment:
does typing the config become a big job? @Natoandro wdyt?

Reply:
Yes. It depends on the runtime on which the type belongs.